nix/pkgs/gomobile/default.nix

```diff
@@ -64,11 +73,17 @@ buildGo122Module {
     ln -s $src $out/src/golang.org/x/mobile
   '';
 
+  # to fix : error: unable to find sdk: 'iphoneos'
```

Comment:
This comment is bad, we don't know what it refers to.

Reply:
Ah sorry I missed this, I was trying to be concise. 
The full error was 
```
 > /nix/store/5c4b1rmbak441s22150gb99crs0jppgb-gomobile-0-unstable-2024-12-13/bin/
gomobile: xcrun --find: exit status 255
> error: unable to find sdk: 'iphoneos'
```